src/layout/header/Header.tsx

```diff
@@ -76,16 +74,17 @@ export const Header = () => {
           to={UrlPathTemplate.Home}>
           <SearchIcon fontSize="large" />
         </MenuIconButton>
-
         {user?.isCreator && (
-          <Button
+          <MenuButton
             sx={{
               gridArea: 'new-result',
               fontSize: '1rem',
               fontWeight: 700,
               gap: '0.5rem',
               display: { xs: 'none', sm: 'inline-flex' },
+              textDecoration: 'none',
             }}
+            isSelected={currentPath.startsWith(UrlPathTemplate.RegistrationNew)}
```

Comment:
Nå vises denne linjen også om man går inn på Landing Page for alle registreringer. 
![Screenshot 2023-12-19 at 14 52 05](https://github.com/BIBSYSDEV/NVA-Frontend/assets/53341294/0dabf2e4-7649-4c54-ad44-39e207415596)

Reply:
Antar kanskje at vi bare ønsker å vise denne som selected når man _skal_ opprette nytt resultat, altså på `/registration`. Tror det blir litt rart å vise den som selected når man åpner wizard også 🤔